spec/onlyoffice_file_helper/file_helper_get_file_size_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe OnlyofficeFileHelper::FileHelper, '.get_file_size' do
+  let(:file_path) { '/tmp/file_size.ext' }
+  let(:file_path_size_nil) { '/tmp/file_size0.ext' }
+
+  before do
+    described_class.create_file_with_size(file_path: file_path, size: '15M')
+  end
+
+  it 'get file size' do
+    expect(described_class.file_size(file_path)).to be(15_728_640)
+  end
+
+  it 'get file size if file size 0' do
```

Comment:
I think another test will be good - what will happen if we send to this method nonexisting file